packages/paste-website/src/pages/patterns/filter-group/index.mdx

```diff
@@ -0,0 +1,278 @@
+- When using the search input field, users can submit a search by hitting the “enter” key. If the search input is for live filtering, results should update immediately upon load without hitting enter, while being mindful of performance.
+
+Always consider opportunities to reduce cognitive load by limiting the number of filters presented. For the simple table filter group we recommend including no more than 5 filters. If you need to include more than 5 filters, please refer to the [Complex table filtering](#complex-table-filtering) variation.
+
+#### Default state
+
+<Img fluid={props.data.defaultState.childImageSharp.fluid} />
+
+#### Empty state
+
+When the applied filters and/or search does not return any results, use the [empty state pattern](/patterns/empty-state) to inform the user and provide a method to reset all filter and search criteria.
+
+<Img fluid={props.data.emptyState.childImageSharp.fluid} />
+
+#### Custom date range
+
+While we don’t yet have an official custom date/time component, in the interim it is recommended to use the following combination of Popover, Date Input, and Time Input to allow customers to select a custom date/time range.
+
+When the Custom option is selected, the Popover containing the Date Inputs and Time Inputs necessary would be displayed to allow customers to enter their desired range. Once the Apply button is clicked, the Popover would hide. If a customer wants to change their original selections, they must click on the Calendar link style button to open the Popover again.
+
+<Img fluid={props.data.customDate.childImageSharp.fluid} />
+
+#### Dos and Don'ts
+
+<DoDont>
+  <Do title="Do" body="Include up to 5 filters in the simple table filter group." />
+  <Dont title="Don't" body="Don’t Include more than 5 filters in the simple table filter group." />
+</DoDont>
+
+<DoDont>
+  <Do
+    title="Do"
+    body="The search input should utilize the placeholder copy to make it very obvious to users what they should enter into the field to search."
+  />
+  <Dont title="Don't" body="Don’t use vague placeholder copy in the search field." />
+</DoDont>
+
+<DoDont>
+  <Do
+    title="Do"
+    body="The search input should be labeled directly using aria-label, so it’s accessible. By doing this, it’s not necessary to include a visible label."
+  />
+  <Dont title="Don't" body="Don’t add a visible label to the search input." />
+</DoDont>
+
+### Complex table filtering
+
+**Complex table filtering** is for users who are adding more than 5 filter parameters, multi-select filter criteria,
+selecting custom query rules, and/or creating complex if/else logic without code to filter down a large data table.
+
+<Callout>
+  <CalloutTitle>This pattern is coming soon!</CalloutTitle>
+  <CalloutText>
+    We've not created this variant of filter group pattern. If you need this for your current or upcoming work, please
+    consider partnering with us to [contribute it](/introduction/contributing/patterns/).
```

Comment:
Issue: the links in these Callouts aren't working. I think if you want to put links inside of a component in mdx, you have to do it w/ the Anchor component.